crp_APL/interfaces/plannerInterfaces/plannerInterfaces.hpp

```diff
@@ -0,0 +1,135 @@
+#ifndef CRP_APL_PLANNER_INTERFACES_INTERFACES_HPP
+#define CRP_APL_PLANNER_INTERFACES_INTERFACES_HPP
+
+#include <vector>
+#include <cstdint>
+
+
+namespace crp
+{
+namespace apl
+{
+
+struct Vector2D
+{
+    float x{0.0f};
+    float y{0.0f};
+};
+
+struct Vector3D
+{
+    float x{0.0f};
+    float y{0.0f};
+    float z{0.0f};
+};
+
+typedef Vector2D Point2D;
+typedef Vector3D Point3D;
+
+struct Pose2D
+{
+    Point2D position;
+    float theta{0.0f};
+};
+
+struct TrajectoryPoint
+{
+    Pose2D pose;
+    float velocity{0.0f};
+};
+
+typedef std::vector<TrajectoryPoint> PlannerOutputTrajectory;
+typedef std::vector<std::vector<float>> OccupancyGrid;
+
+struct PlannerInputPose2D
+{
+    Pose2D pose;
+    float kappa {0.0f}; // road curvature in 1/m
```

Comment:
Maybe it is a takeover from my code, but I'd rather use road curvature, or curvature simply instead of kappa, it is more intuitive

Reply:
or, rather not include curvature in a 'pose' type